packages/named-locks/src/index.ts

```diff
@@ -95,7 +96,7 @@ export async function close() {
  * @param name The name of the lock to acquire.
  */
 export async function tryLockAsync(name: string): Promise<Lock | null> {
-  return getLock(name, { wait: false });
+  return getLock(name, { timeout: 0 });
```

Comment:
I simplified this a bit by making `timeout: 0` imply "don't wait"; any non-zero timeout means we should in fact wait.